tests/test_callback.py

```diff
@@ -37,6 +37,10 @@ def test_callback(tmpdir, vcrmode):
         contents = real_file.read()
         contents = re.sub(r"\d+-\d+-\d+[ T]\d+:\d+:\d+\.\d+", "2000-01-01 12:00:00.0000", contents)
         contents = re.sub(r"\d+:\d+:\d+\.\d+", "12:00:00.0000", contents)
+        # cover differencess across Ansible versions for the legacy Foreman output
+        # contents = re.sub(r"\\\"warn\\\": (true|false)", '\\"warn\\": false', contents)
```

Comment:
Tests are green, hopefully, it works this time. Any other comments before I remove this line?

Reply:
This commented one? Nah, away with it.